packages/components/chip/src/Chip.styles.tsx

```diff
@@ -0,0 +1,183 @@
+       *
+       * - `tinted`: Chip will be filled but using a lighter color scheme.
+       *
+       * - `dashed`: Chip will be transparent with an outline dashed.
+       */
+      design: makeVariants<'design', ['filled', 'outlined', 'tinted', 'dashed']>({
+        filled: [],
+        outlined: ['bg-transparent', 'ring-1', 'ring-current'],
+        tinted: [],
+        dashed: [
+          'bg-transparent border-sm border-dashed shadow-none focus-visible:border-outline-high',
+        ],
+      }),
+      /**
+       * Color scheme of the chip.
+       */
+      intent: makeVariants<
+        'intent',
+        ['primary', 'secondary', 'success', 'alert', 'danger', 'info', 'neutral', 'surface']
+      >({
+        primary: [],
+        secondary: [],
+        success: [],
+        alert: [],
+        danger: [],
+        info: [],
+        neutral: [],
+        surface: [],
+      }),
+      /**
+       * Disable the chip, preventing user interaction and adding opacity.
+       */
+      disabled: {
+        true: ['cursor-not-allowed', 'opacity-dim-3'],
+      },
+    },
+    compoundVariants: [
+      ...filledVariants,
+      ...outlinedVariants,
+      ...tintedVariants,
+      ...dashedVariants,
+    ],
+    defaultVariants: {
+      design: 'outlined',
+      intent: 'primary',
+    },
+  }
+)
+
+export const chipContentStyles = cva(
+  ['gap-sm inline-flex h-full items-center justify-center whitespace-nowrap'],
+  {
+    variants: {
+      isBordered: {
+        no: ['max-w-[240px]'],
+        yes: ['max-w-[238px]'],
+      },
+      mode: {
+        default: [],
+        icon: [],
+      },
+      hasCloseIcon: {
+        yes: [],
+        no: [],
+      },
+    },
+    defaultVariants: {
+      isBordered: 'no',
+      mode: 'default',
+      hasCloseIcon: 'no',
+    },
+    compoundVariants: [
+      {
+        mode: 'default',
+        hasCloseIcon: 'no',
+        isBordered: 'no',
+        class: tw(['px-md']),
+      },
+      {
+        mode: 'default',
+        hasCloseIcon: 'no',
+        isBordered: 'yes',
+        class: tw(['px-[7px]']),
+      },
+      {
+        mode: 'icon',
+        hasCloseIcon: 'no',
+        isBordered: 'no',
+        class: tw(['px-md']),
+      },
+      {
+        mode: 'icon',
+        hasCloseIcon: 'no',
+        isBordered: 'yes',
+        class: tw(['px-md']),
+      },
+      {
+        mode: 'default',
+        hasCloseIcon: 'yes',
+        isBordered: 'no',
+        class: tw(['pl-md']),
+      },
+      {
+        mode: 'default',
+        hasCloseIcon: 'yes',
+        isBordered: 'yes',
+        class: tw(['pl-[7px]']),
+      },
+      {
+        mode: 'icon',
+        hasCloseIcon: 'yes',
+        isBordered: 'no',
+        class: tw(['pl-md']),
+      },
+      {
+        mode: 'icon',
+        hasCloseIcon: 'yes',
+        isBordered: 'yes',
+        class: tw(['pl-md']),
+      },
+    ],
+  }
+)
+
+export const chipContentTextStyles = cva(['overflow-hidden text-ellipsis'])
```

Comment:
Imho this is the type of things where using `cva` and calling its function is overkill, and we should simply use tailwind classes on the markup.

Or at least we could use `cx()` instead of `cva()` (which is meant to manage variants more than classnames)